units/ArmBuildings/LandUtil/armdrag.lua

```diff
@@ -32,7 +32,7 @@ return {
 		maxwaterdepth = 0,
 		name = unitName,
 		objectname = "Units/armdrag.s3o",
-		repairable = false,
+		repairable = true,
```

Comment:
Consider updating the gadget

https://github.com/beyond-all-reason/Beyond-All-Reason/blob/1291539e0a0b99a6219e6e227182c3e314e9f475/luarules/gadgets/unit_objectify.lua#L19